zerver/lib/email_mirror.py

```diff
@@ -187,6 +187,11 @@ class ZulipEmailForwardError(Exception):
     pass
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
+    if len(topic) > MAX_TOPIC_NAME_LENGTH:
+        try:
+            Message.objects.get(subject=topic[:MAX_TOPIC_NAME_LENGTH])
+        except Message.DoesNotExist:
+            content = '{}\n{}'.format(topic, content)
```

Comment:
Maybe we want a double newline, and `Subject: ` at the start?  I think that'd make the subject stand out better.

Reply:
yeah ill work on it

Reply:
can you pls elaborate on what you mean by "Subject:"